### ISSUE TITLE
App Authentication

### ISSUE THREAD

**Issue by radekcep** [dropbox/SwiftyDropbox#181]
Is it possible to use App Authentication as described here: https://www.dropbox.com/developers/reference/auth-types#app to download from specific user's public folder?

**Comment by greg-db** [comment]
No, to download from a shared link you would use [`getSharedLinkFile`](https://dropbox.github.io/SwiftyDropbox/api-docs/latest/Classes/SharingRoutes.html#/s:FC13SwiftyDropbox13SharingRoutes17getSharedLinkFileFT3urlSS4pathGSqSS_12linkPasswordGSqSS_9overwriteSb11destinationFTV10Foundation3URLCSo15HTTPURLResponse_S2__GCS_19DownloadRequestFileCCS_7Sharing28SharedLinkMetadataSerializerCS5_32GetSharedLinkFileErrorSerializer_), but that doesn't support app authentication unfortunately, only user authentication. I'll pass this along as a feature request for app authentication support on that.
